config/config_test.go

```diff
@@ -617,12 +617,14 @@ func TestWorkingDirectory(t *testing.T) {
 	checkValue(cwd)
 
 	// env override
-	t.Setenv("TREEFMT_WORKING_DIR", "/fizz/buzz/..")
-	checkValue("/fizz")
+	cwd = t.TempDir()
+	t.Setenv("TREEFMT_WORKING_DIR", cwd+"/buzz/..")
+	checkValue(cwd)
 
 	// flag override
-	as.NoError(flags.Set("working-dir", "/flip/flop"))
-	checkValue("/flip/flop")
```

Comment:
I don't understand why this test was changed?

Is it made redundant by the new test?

Reply:
With the new logic I needed these to be real directories.